index/index.html

```diff
@@ -3,67 +3,106 @@
   <head>
     <meta charset="utf-8">
     <title></title>
-    <link rel="stylesheet" type="text/css" media="all" href="css/reset.css" />
-    <link rel="stylesheet" type="text/css" media="all" href="css/text.css" />
-    <link rel="stylesheet" type="text/css" media="all" href="css/960.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="../css/reset.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="../css/text.css" />
+    <link rel="stylesheet" type="text/css" media="all" href="../css/960.css" />
     <link rel="stylesheet" href="style.css" />
     <script
    src="https://code.jquery.com/jquery-3.1.1.js"
    integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA="
    crossorigin="anonymous"></script>
+  <div class="container_16 pull_6">
+   <nav id="nav-bar">
+     <ul>
+       <li><a id="nav-bar-item" href="#">Home</a></li>
```

Comment:
nav-bar-item would be better as a class.  try to reserve ids for javascript hooks, and make sure you only have one element with a specific id per page.  you should only use an id once